lib/features/account/account_page.dart

```diff
@@ -8,7 +8,10 @@ class AccountPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Account')),
+      appBar: AppBar(
+        title: const Text('Account'),
+        scrolledUnderElevation: 0,
+      ),
```

Comment:
@ethan-tbd should we set this at the theme level?
```dart
appBarTheme: AppBarTheme(
          scrolledUnderElevation: 0,
        ),
```

Reply:
done.